### PR TITLE
updating cilium links

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6541)

### CODE REVIEW

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 1]
```diff
@@ -20,20 +20,21 @@ For background on Cilium, read the [Introduction to Cilium](http://cilium.readth
...
 To get familiar with Cilium easily you can follow the
-[Cilium Kubernetes Getting Started Guide](http://www.cilium.io/try)
+[Cilium Kubernetes Getting Started Guide](https://www.cilium.io/try)
 to perform a basic DaemonSet installation of Cilium in minikube.
 
 Installation in a minikube setup uses a simple ''all-in-one'' YAML
-file that includes DaemonSet configurations for Cilium and a key-value store
-(consul) as well as appropriate RBAC settings:
+file that includes DaemonSet configurations for Cilium, to connect
+to a the minikube's etcd instance as well as appropriate RBAC settings:
```
Comment (tengqm) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: a the -> the

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 2]
```diff
@@ -42,7 +43,7 @@ policies, as well as L7 (e.g., HTTP) security policies using an example applicat
 ## Deploying Cilium for Production Use
 
 For detailed instructions around deploying Cilium for production, see:
-[Cilium Administrator Guide](http://cilium.readthedocs.io/en/latest/admin/) This
+[Cilium Administrator Guide](https://cilium.readthedocs.io/en/latest/install/#kubernetes-installation-guide) This
```
Comment (tengqm) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: if the admin guide is gone, we should rename the link to installation guide.

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 3]
```diff
@@ -67,7 +68,7 @@ cilium          1         1         1         <none>          2m
...
 
 - One `cilium` Pod runs on each node in your cluster and enforces network policy on the traffic to/from Pods on that node using Linux BPF.
-- For production deployments, Cilium should leverage the key-value store cluster (e.g., etcd) used by Kubernetes, which typically runs on the Kubernetes master nodes.   The [Cilium Administrator Guide](http://cilium.readthedocs.io/en/latest/admin/) includes an example DaemonSet which can be customized to point to this key-value store cluster.   The simple ''all-in-one'' DaemonSet for minikube requires no such configuration because it automatically deploys a `cilium-consul` Pod to provide a key-value store.
+- For production deployments, Cilium should leverage the key-value store cluster (e.g., etcd) used by Kubernetes, which typically runs on the Kubernetes master nodes.   The [Cilium Administrator Guide](https://cilium.readthedocs.io/en/latest/install/#kubernetes-installation-guide) includes an example DaemonSet which can be customized to point to this key-value store cluster.   The simple ''all-in-one'' DaemonSet for minikube requires no such configuration because it automatically connects to the minikube's etcd instance.
```
Comment (tengqm) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: if the admin guide is gone, we should rename the link to installation guide.

Comment (tengqm) [reply docs/tasks/administer-cluster/cilium-network-policy.md]: use single space between sentences.

File: docs/tasks/administer-cluster/cilium-network-policy.md (hunk 1 above)
Comment (aanm) [reply docs/tasks/administer-cluster/cilium-network-policy.md]: Addressed

File: docs/tasks/administer-cluster/cilium-network-policy.md (hunk 3 above)
Comment (aanm) [reply docs/tasks/administer-cluster/cilium-network-policy.md]: Addressed

File: docs/tasks/administer-cluster/cilium-network-policy.md (hunk 2 above)
Comment (aanm) [reply docs/tasks/administer-cluster/cilium-network-policy.md]: Addressed

**Comment by aanm** [comment]
@a-mccarthy can you merge this please? thank you

**Comment by heckj** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/website/6541)*

**Comment by heckj** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/website/6541#-:-L2HZg98m8K1S8kzucZ_:bnfp4nl)*

**Pull request merged**